packages/base/src/dialogs/terrainDialog.tsx

```diff
@@ -80,8 +80,6 @@ const TerrainDialog = ({
         className="jp-mod-styled"
         type="number"
         min={0}
-        max={1}
-        step={0.1}
```

Comment:
We should leave the step, otherwise the `ok` button gets disabled on non-integers.